src/main/java/org/apache/commons/lang3/ClassUtils.java

```diff
@@ -935,6 +935,60 @@ public static boolean isAssignable(Class<?> cls, final Class<?> toClass, final b
         return toClass.isAssignableFrom(cls);
     }
 
+    /**
+     * <p>Check if the given {@code Method} is overridable in the given target {@code Class}.</p>
+     *
+     * <p>This method returns {@code false} for any {@code null} input or method with the static or final modifiers.</p>
+     * <pre>
+     * <code>
+     *   private static class ABG {
+     *        private void privateMethod(){
```

Comment:
Add whitespace before the curly brace `{` (to match our coding style)

Reply:
Done